addons/base_automation/models/base_automation.py

```diff
@@ -594,6 +594,15 @@ def _get_eval_context(self, payload=None):
             eval_context['payload'] = payload
         return eval_context
 
+    def _get_evaluated_domain(self, domain):
+        self.ensure_one()
+        domain = domain.replace('.to_utc()', '')
+        return safe_eval.safe_eval(domain, {
+            **self._get_eval_context(),
+            'context_today': safe_eval.datetime.datetime.today,
```

Comment:
Possibly wrong place. This would mean that this function is supported only in base_automation, but domains are not limited to that.